go-runtime/ftl/ftltest/ftltest.go

```diff
@@ -25,6 +25,9 @@ import (
 	mcu "github.com/TBD54566975/ftl/internal/testutils/modulecontext"
 )
 
+// Allows tests to mock module reflection
+var moduleGetter = reflection.Module
```

Comment:
We probably want to eventually figure out a more comprehensive way to stub out the reflection for tests.